wallet/wallet.go

```diff
@@ -5895,6 +5898,7 @@ func (w *Wallet) ForUnspentUnexpiredTickets(ctx context.Context,
 			ticketHash := *ticketSummary.Ticket.Hash
 			err := f(&ticketHash)
 			if err != nil {
+				errs = errors.Join(errs, err)
```

Comment:
this will create an error chain of many errors that each unwrap to the next error, but instead we want all of them to be returned by a single Unwrap() []error call.

I would do `var errs []error` and then `errors.Join(errs...)` to do the final join before returning.  This also works if the slice is empty, or some of the errors in the slice are nil (they are just ignored).

Reply:
Done